### PR TITLE
【fix issue#1270】Update NNI Introduction Board 

### PR DESCRIPTION
**Pull request by rabbit008**
1.Adjust the table
2.fixed some broken links
It's not ready. there are two links that cannot be clicked until two related documents are merged（pr#1401 pr#1407）

### CODE REVIEW

**Comment by QuanluZhang** [comment]
I think we still need a meeting to discuss the content.

File: README.md [hunk 1]
```diff
@@ -291,4 +337,4 @@ Now we have some external usage examples run in NNI from our contributors. Thank
 
 ## **License**
 
-The entire codebase is under [MIT license](LICENSE)
+
```
Comment (xuehui1991) [inline README.md]: why remove this license?

Comment (rabbit008) [reply README.md]: Incorrect operation,fixed

File: README.md [hunk 2]
```diff
@@ -27,8 +27,10 @@ The tool dispatches and runs trial jobs generated by tuning algorithms to search
 <table>
   <tbody>
     <tr align="center" valign="bottom">
+    <td>
+      </td>
       <td>
-        <b>Supported Frameworks</b>
+        <b>Supported</b>
```
Comment (scarlett2018) [inline README.md]: let's keep the "Supported Frameworks" there, "supported" is not better than the previous version.

Comment (rabbit008) [reply README.md]: change to Frameworks&Libraries

**Pull request merged**